application/config.py

```diff
@@ -0,0 +1,15 @@
+import pyodbc
+import sys, os
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import keys
+
```

Comment:
You can remove some newlines i  between here too

Reply:
agree